### DISCUSSION TITLE
How to change Jupyter startup directory?

### DISCUSSION THREAD

**Discussion by WolframStrauss** [winpython/winpython#984] [Q&A]
Hi,
I just installed version 3.9.4 on a new computer. On my old machine, I could change the Jupyter startup dirctory to my user home folder.
I tried the following:
1. Set in `.jupyter/jupyter_notebook_config.py` variable `c.NotebookApp.notebook_dir` to my user home directory.
1. Copy of `jupyter_notebook_config.py` to `..WPy64-3940\settings\.jupyter`
1. Uncomment the following entries in `winpython.ini`
	- USERPROFILE = %HOME%
	- JUPYTER_DATA_DIR = %HOME%
	- JUPYTERLAB_SETTINGS_DIR = %HOME%\.jupyter\lab
	- JUPYTERLAB_WORKSPACES_DIR = %HOME%\.jupyter\lab\workspaces

So far no luck.

Any ideas are wellcome.

Thank you. Wolfram

**Comment by thomas-haslwanter** [comment]
I assume you are using Windows.
The easiest thing is to generate a file "ipynb.bat', somewhere in your PATH (e.g. in your %HOME%), containing
`cd <startup_dir> & jupyter lab`
